packages/static-config/test/index.test.ts

```diff
@@ -16,6 +16,30 @@ describe('getConfig()', () => {
     `);
   });
 
+  it('should parse named export config from Node.js file', () => {
+    const project = new Project();
+    const sourcePath = join(__dirname, 'fixtures/node-named-config.js');
+    const config = getConfig(project, sourcePath);
+    expect(config).toMatchInlineSnapshot(`
+      {
+        "maxDuration": 30,
+      }
+    `);
+  });
+
+  it('should prioritize config export over named export from Node.js file', () => {
+    const project = new Project();
+    const sourcePath = join(__dirname, 'fixtures/node-duplicate-config.js');
+    const config = getConfig(project, sourcePath);
+    expect(config).toMatchInlineSnapshot(`
+      {
+        "maxDuration": 60,
```

Comment:
Bikesheddy question: Is this a design decision that `const config` take precedence over `const maxDuration`? The latter looks more explicit and is intuitive to me. I don't have strong opinion here btw 😄

Reply:
> Bikesheddy question: Is this a design decision that `const config` take precedence over `const maxDuration`? The latter looks more explicit and is intuitive to me. I don't have strong opinion here btw 😄

My strong opinion is that we should error if people try to do it twice, but that would not be in sync with what we have with `export const config` and `vercel.json`. In those cases the `vercel.json` values have higher priority. I think we can revisit this at a later stage when reworking this entire behavior?

Reply:
I think it should error. `vercel.json` isn't exactly the same case, since it specifies a glob pattern, whereas the in-code config is specific to that specific function, so declaring configuration twice would seem like an error to me.

We should make a decision on this now since it would be a breaking change if we wanted to change it in the future.

Reply:
Thanks for your input Nate. I’ll chat about this with Javi tomorrow, but most likely changing this to error then.